src/TitleDetails/BoundItemsList/hooks/useItemsList/useItemsList.js

```diff
@@ -0,0 +1,56 @@
+import { useQuery } from 'react-query';
+
+import {
+  batchRequest,
+  ITEMS_API,
+  LIMIT_MAX,
+  ORDER_PIECES_API,
+} from '@folio/stripes-acq-components';
+import {
+  useNamespace,
+  useOkapiKy,
+} from '@folio/stripes/core';
+
+const DEFAULT_DATA = [];
+
+export const useItemsList = ({ titleId, poLineId, options = {} }) => {
```

Comment:
I would suggest using a name like `useBoundItems` because the current name is too abstract in comparison with the actual functionality.

Reply:
updated